mimic/model/lstm_model.py

```diff
@@ -0,0 +1,80 @@
+from mimic.model.model import Model
+
+from keras.preprocessing.sequence import pad_sequences
```

Comment:
Keras and TensorFlow will have to be added to the Pipfile once #14 is merged.

Reply:
I just merged #14, so you'll want to do this now. I would recommend rebasing on the latest version of master. In particular, in case you don't know how to do that (I should really have all of Launch Pad go through a rebase course just in case...):
* If you have any changes left on your local copy of this branch, commit, stash, or destroy them.
* Checkout out master and pull the latest version.
* Go back to this branch and do `git rebase master`.
* There shouldn't be any merge conflicts, but if there are, you can fix them, `git add` them, then `git rebase --continue` (you may have to do this repeatedly).
* You can now add more changes to the latest version of the branch!
* Finally, when you push, you'll have to do `git push -f` since the history of the branch has changed.

Reply:
In particular, you'll want to add Keras and TensorFlow under `[packages]` in `Pipfile`, then run `pipenv update --dev` to install all the dependencies and generate a new `Pipfile.lock`, which should be committed alongside `Pipfile` (there's slightly more detail in the [README](https://github.com/ubclaunchpad/mimic/blob/master/README.md)).